pyimzml/ImzMLWriter.py

```diff
@@ -105,16 +106,25 @@
 """
 
 class ImzMLWriter(object):
-    def __init__(self, output_filename, mz_dtype=np.float64, intensity_dtype=np.float32):
+    def __init__(self, output_filename, mz_dtype=np.float64, intensity_dtype=np.float32, mz_compression=False, intensity_compression=False):
+        '''"output_filename" is used to make the base name by removing the extension (if any).
+        two files will be made by adding ".ibd" and ".imzML" to the base name
+        processed or continuous mode will be chosen automatically. Identical mz lists will only be written once.
+        Force process mode by setting "instance.hashes = None"
+        "compression" must be True or False or indicate a number to round to.
```

Comment:
I don't think any more compression options will be added; but since room was left in the standard I'll agree that I should leave room for it here too.

I don't like the idea of classes since compression can be applied to the mz OR the intensities. It is important not to force both or none, especially when using rounding, since people may be willing to lose precision on the intensities for the sake of filesize, but unwilling to lose precision on the m/zs.

What do you think of a parseable parameter with <compression type><space><#decimals>: `mz_compression=None|"zlib"|"zlib 2"`?